addons/project/tests/test_project_sharing_portal_access.py

```diff
@@ -4,8 +4,11 @@
 from collections import OrderedDict
 from lxml import etree
 from odoo import Command
+from odoo.addons.portal.controllers.mail import PortalChatter
+from odoo.addons.website.tools import MockRequest
```

Comment:
We cannot use `MockRequest` since `website` module is not in the dependencies of project.